HOWTO/xero.md

```diff
@@ -0,0 +1,71 @@
+#XERO
+
+##How to setup invoice template
+
+1. From the top menu navigate to **Settings > General Settings**
+2. From section **Features** navigate to **Invoice Settings**
+3. Pick an invoice to copy, from **Options** select **Copy**, set a name to identify invoice for your event, eg [Your event name, year]
+4. Customise your invoice:
+* Select **Options > Change Logo**
+Upload your image (maximum of 400 pixels wide and 120 pixels high).
+* Select **Options > Edit**
+Review selected fields (most of the fields/options should be preselected from the copied invoice). 
+
+Update the following fields:
+
+* **"Terms & Payment Advice (Invoice)"**
+Include your payment details, eg bank account, payment terms, etc. Sample:
+
+```
+Electronic payment preferred:
+Please, include invoice number in all electronic transfers.
+Please, email electronic transaction confirmation details to [your@mail.account]
+
+Direct deposit:
+  Bank:             [Your bank name] 
```

Comment:
I think we should make this clear that it's the bank details provided by Linux Australia. Don't want conferences accidentally using their personal banks or thinking opening a new account is necessary by the subcommittee.